### ISSUE TITLE
Conflict with Advanced Custom Fields Plugin

### ISSUE THREAD

**Issue by tmconnect** [wp-shortcake/shortcake#90]
If using both plugins together I got two javascript errors using the ACF field settings

[Error] TypeError: undefined is not an object (evaluating 'wp.media.view.settings.post.id')
    (anonyme Funktion) (load-scripts.php, line 157)
    global code (load-scripts.php, line 157)
[Error] TypeError: undefined is not an object (evaluating 'wp.mce.views')
    (anonyme Funktion) (shortcode-ui.js, line 1074)
    forEach ([native code], line 0)
    forEach (load-scripts.php, line 54)
    (anonyme Funktion) (load-scripts.php, line 55)
    (anonyme Funktion) (shortcode-ui.js, line 1070)
    j (load-scripts.php, line 2)
    fireWith (load-scripts.php, line 2)
    ready (load-scripts.php, line 2)
    J (load-scripts.php, line 2)

**Comment by coderaaron** [comment]
Here is the quick fix I use:

change 

```
        sui.shortcodes.each( function( shortcode ) {
            // Register the mce view for each shortcode.
            // Note - clone the constructor.
            wp.mce.views.register(
                shortcode.get('shortcode_tag'),
                $.extend( true, {}, sui.utils.shortcodeViewConstructor )
            );
        } );
```

to

```
        sui.shortcodes.each( function( shortcode ) {
            if( wp.mce ) {
                // Register the mce view for each shortcode.
                // Note - clone the constructor.
                wp.mce.views.register(
                    shortcode.get('shortcode_tag'),
                    $.extend( true, {}, sui.utils.shortcodeViewConstructor )
                );
            }
        } );
```

starting at line 1070 in shortcode-ui.js

Be warned I haven't done extensive testing with this "hack", but it seems to work for what I've been doing.

**Comment by danielbachhuber** [comment]
Fixed in #118
